x-pack/plugins/synthetics/public/apps/synthetics/components/settings/private_locations/location_form.tsx

```diff
@@ -131,8 +100,41 @@ export const LocationForm = ({
               }
             </p>
           </EuiCallOut>
-        )}
-      </EuiForm>
+
+          <EuiSpacer />
+          {selectedPolicy?.agents === 0 && (
+            <EuiCallOut
+              title={AGENT_MISSING_CALLOUT_TITLE}
+              size="s"
+              style={{ textAlign: 'left' }}
+              color="warning"
+            >
+              <p>
+                {
+                  <FormattedMessage
+                    id="xpack.synthetics.monitorManagement.agentMissingCallout.content"
+                    defaultMessage="You have selected an agent policy that has no agents attached. Please ensure you have at least one agent enrolled in this policy. You can add agent before or after creating a location. For more information, {link}."
```

Comment:
```suggestion
                    defaultMessage="You have selected an agent policy that has no agent attached. Make sure that you have at least one agent enrolled in this policy. You can add agent before or after creating a location. For more information, {link}."
```

Putting agent in singular because a few lines above in the form we recommend using a single agent per policy